lib/challange4.rb

```diff
@@ -0,0 +1,20 @@
+def game(number, tries)
+  guess = gets.to_i
+  if guess > number
+    puts "Number is lower than #{guess}"
+    tries += 1
+  elsif guess < number
+    puts "Number is higher than #{guess}"
+    tries += 1
+  elsif guess == number
+    puts "CORRECT! The number is #{number}"
+    puts "It took you #{tries} tries"
+    return
+  end
+  game(number, tries)
```

Comment:
Good job calling the function from with in itself rather than using a conditional loop.